### ISSUE TITLE
Packaging problem with special character

### ISSUE THREAD

**Issue by EsqualiburDEV** [airsdk/Adobe-Runtime-Support#1704]
## Problem Description

Hello
I need to include folders with files in my apk.
folder path: myproject\__install

In "adt.log", I have:
ADT 33.1.1.779 called with: -package -target apk-captive-runtime -storetype PKCS12 -keystore C:\Users\User\AppData\Local\JetBrains\Toolbox\apps\IDEA-U\ch-0\213.6777.52. plugins\flex\lib\temp_keystore.p12 -storepass xxxxxx E:\Repo_Git\MyProject\bin\MyProject.apk E:\Repo_Git\MyProject\bin\MyProject-app.xml -extdir E:\Repo_Git\MyProject\ane -CE :\Repo_Git\MyProject\bin MyProject.swf -CE:\Repo_Git\MyProject\src assets -CE:\Repo_Git\MyProject\src __install

The "__install" folder is not integrated into the "assets" directory of the APK.

When I rename the "__install" folder to "install", the folder is embedded.

I am using the new system to package and generate the apk.
With the old system, this was not a problem (until 33.1.1.633 include).

AIR SDK : 33.1.1.779
Affected environment : Android (APK)

## Steps to Reproduce
- rename a folder you want to integrate (eg __install)
- include it in the packaging: -C E:\Repo_Git\edl\EDLWai\src\__install
- open the "APK" file
- find that the file is not included

## Known Workarounds
Rename the folder without special "_" characters.
But for some projects (especially old projects) this can be problematic.

**Comment by itlancer** [comment]
Similar to https://github.com/airsdk/Adobe-Runtime-Support/issues/1188

**Comment by ajwfrost** [comment]
This is default Android Studio behaviour (or Android Gradle Plug-in). There are a set of files that are ignored by default:
https://android.googlesource.com/platform/frameworks/base/+/b41af58f49d371cedf041443d20a1893f7f6c840/tools/aapt/AaptAssets.cpp#60
```
"!.svn:!.git:!.ds_store:!*.scc:.*:<dir>_*:!CVS:!thumbs.db:!picasa.ini:!*~"
```
and this includes `<dir>_*` i.e. any directory that has an underscore followed by anything.

Given the previous behaviour of AIR SDK was to include these anyway, I think we should be able to just override this in the gradle file so that these folders are then packaged... 

thanks

**Comment by EsqualiburDEV** [comment]
@itlancer: I don't think so, it's just the inclusion in the package of a folder of static files.
No native extension creation.

@ajwfrost: Thank you :)

**Comment by EsqualiburDEV** [comment]
thank you, the problem is solved in the last version AIR version 33.1.1.795
